modules/ztp-retrieving-the-host-firmware-settings.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-retrieving-the-host-firmware-settings_{context}"]
+= Retrieving the host firmware settings for a managed cluster
+
+You can retrieve the host firmware settings for managed clusters.
+This is useful when you have deployed changes to the host firmware and you want to monitor that the changes and ensure that they are applied successfully.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed {rh-rhacm-first} and logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have provisioned a cluster that is managed by {rh-rhacm}.
+
+.Procedure
+
+. Retrieve the host firmware settings for the managed cluster.
+Run the following command:
++
+--
+[source,terminal]
+----
+$ oc get hostfirmwaresettings -n <managed_cluster_namespace> -o yaml
+----
+
+.Example output
+[source,terminal]
+----
+apiVersion: v1
+items:
+- apiVersion: metal3.io/v1alpha1
+  kind: HostFirmwareSettings
+  metadata:
+    creationTimestamp: "2024-09-11T10:29:43Z"
+    generation: 1
+    name: compute-1.example.com
+    namespace: kni-qe-24
+    ownerReferences:
+    - apiVersion: metal3.io/v1alpha1
+      blockOwnerDeletion: true
+      controller: true
+      kind: BareMetalHost
+      name: compute-1.example.com
+      uid: 0baddbb7-bb34-4224-8427-3d01d91c9287
+    resourceVersion: "280057626"
+    uid: 65d0e89b-1cd8-4317-966d-2fbbbe033fe9
+  spec:
+    settings: {}
+  status:
+    conditions:
+    - lastTransitionTime: "2024-09-11T10:29:43Z"
+      message: ""
+      observedGeneration: 1
+      reason: Success
+      status: "True" <1>
+      type: ChangeDetected
+    - lastTransitionTime: "2024-09-11T10:29:43Z"
+      message: Invalid BIOS setting
+      observedGeneration: 1
+      reason: ConfigurationError
+      status: "False" <2>
+      type: Valid
+    lastUpdated: "2024-09-11T10:29:43Z"
+    schema:
+      name: schema-40562318
+      namespace: compute-1
+    settings: <3>
+      AccessControlService: Enabled
+      AcpiHpet: Enabled
+      AcpiRootBridgePxm: Enabled
+      # ...
+----
+<1> Indicates that the host firmware has changed
```

Comment:
To make it more clear: This status indicates that a change in the host firmware settings has been detected.